Testing/ORKTest/ORKTest/ORKESerialization.m

```diff
@@ -1085,6 +1095,13 @@ @implementation ORKESerializer
             PROPERTY(numericAnswer, NSNumber, NSObject, NO, nil, nil),
             PROPERTY(unit, NSString, NSObject, NO, nil, nil)
             })),
+   ENTRY(ORKLocationQuestionResult,
+         nil,
+         (@{
+            PROPERTY(locationAnswer, MKPlacemark, NSObject, NO,
+                     ^id(id placemark) { return [placemark ork_JSONDictionary]; },
```

Comment:
Somehow the implementation of `ork_JSONDictionary` is disappeared from this patch.